modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,166 @@ The properties for broker-wide throughput quota balancing are configured at the
+. Default quota configured through the Kafka API on `client_id`
+. Default quota configured through cluster configuration properties on `client_id`
 
-NOTE: The client throughput quotas limit the rates within each shard (logical CPU core) of a Redpanda broker's node. The quotas are neither shared nor balanced between shards and brokers.
+Redpanda Data recommends <<migrate,migrating>> over from cluster configuration-managed quotas to managing the quotas through the Kafka API. You can recreate the configuration-based quotas through the Kafka API using `rpk`, and then remove the quota cluster configs.
+
+You can also use Redpanda Console to view client quotas that are enforced.
 
 === Individual client throughput limit
 
-To set throughput quota for a single client, configure the cluster properties xref:reference:cluster-properties.adoc#target_quota_byte_rate[`target_quota_byte_rate`] and xref:reference:cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`].
+You can view current throughput quotas set through the Kafka API by running the `rpk cluster quotas describe` command.
+
+For example, to see the quotas for client ID `consumer-1`:
 
-* The `target_quota_byte_rate` property applies to a producer client that isn't a member of a client group configured by xref:reference:cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`]. It sets the maximum throughput quota of a client sending to a Redpanda broker.
-* The `target_fetch_quota_byte_rate` property applies to a consumer client that isn't a member of a client group configured by xref:reference:cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`]. It sets the maximum throughput quota of a client fetching from a Redpanda broker.
+[,bash]
+----
+rpk cluster quotas describe --name client-id=consumer-1
+----
 
-The values of both `target_quota_byte_rate` and `target_fetch_quota_byte_rate` are throughput rate limits within a shard, in bytes per second.
+To set a throughput quota for a single client, use the `rpk cluster quotas alter` command. 
+
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id=consumer-1
+----
 
 === Group of clients throughput limit
 
-To set throughput quota for a group of clients, configure the cluster properties xref:reference:cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`] and xref:reference:cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`].
+Alternatively, you can view or configure throughput quotas for a group of clients based on a match on client ID prefix. The following example sets the `consumer_byte_rate` quota to a group of clients that have IDs starting with `consumer-`:
```

Comment:
Not sure--is "starting with `consumer` different than "client IDs prefixed with `consumer-`"? Maybe just used "prefixed"?